keps/sig-node/4191-split-image-filesystem/README.md

```diff
@@ -0,0 +1,923 @@
+- ImageFs monitors read-only
+
+Eviction manager decides the priority of eviction based on which filesystem is experencing pressure.
+
+If Node filesystem experiences pressure, ranking is done as `local volumes + logs of all containers + writeable layer of all containers`
+
+If Image filesystem experiences pressure, ranking is done as `storage of images`.
+
+### Test Plan
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+- (pkg/kubelet/eviction): Sep 11th 2023 - 69.9
+- (pkg/kubelet/stats): Sep 11th 2023 - 77.9
+- (pkg/kubelet/server/stat): Sep 11th 2023 - 55
+
+This KEP will need to cover unit tests where dedicatedImageFs is true.  
+There is currently little test coverage when a separate imagefs is used.  [Issue-120061](https://github.com/kubernetes/kubernetes/issues/120061) has been created to help resolve this.
+
+##### Integration tests
+
+Typically these type of tests are done with e2e tests.
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+This code affects stats, eviction and the summary API.  
+
+There should be e2e tests for each of these components with a split disk.  
+However, there are a few complications with this goal.
+
+1. E2E tests around eviction with a single disk are currently [containerd-eviction](https://testgrid.k8s.io/sig-node-cri-o#ci-CRI-O-cgroupv1-node-e2e-eviction) and [CRI-O-eviction](https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-eviction) failing.  
```

Comment:
nit: the links versus text appear to be swapped? 🙃  (containerd text => cri-o testgrid link and vice versa)

Reply:
Oops.